### ISSUE TITLE
Missing ease as function in docs

### ISSUE THREAD

**Issue by LarsAsDev** [framer/motion#667]
I am trying to make a bounce out effect. But i cannot find a way to make this happen.

I have found a [continous bouncing effect](https://blog.sethcorker.com/framer-motion-bouncing-ball-animation), but it needs to ease out.
There is also [an example](https://spectrum.chat/framer/general/physics-based-animations-like-a-bouncing-ball-or-throwing-objects-in-framer~22056a59-ee4a-4a8e-832a-195b17895d25) that slows down after x seconds, but it doesnt looks realistic.

I think a solution like the type [spring](https://www.framer.com/api/motion/types/#spring) would work for a bounce effect.

**Comment by InventingWithMonster** [comment]
Hey there

`ease` accepts custom easing functions. You can find a bouncing ease function here: https://easings.net/#easeOutBounce

**Comment by LarsAsDev** [comment]
I didn't know it was possible to pass a function via the ease parameter. 
I did not find it in the [Motion documentation](https://www.framer.com/api/motion/types/#tween.ease), but it was mentioned in the [Framer documentation](https://www.framer.com/api/animation/#easingfunction). 

I have just started converting my code from gsap to Framer Motion, so I have a few things to learn. 
In any case, it is very nice to create animation in React with Framer Motion, thanks for your work.
